grants_tagger/preprocess.py

```diff
@@ -34,14 +34,24 @@ def yield_preprocess_data(
     # cols = ['Grant Team', 'ERG', 'Lead Applicant', 'Organisation', 'Scheme', 'Title', 'Synopsis', 'Lay Summary', 'Qu.']
     processed_data = data.drop_duplicates(subset=["Grant_ID", "Sciencetags"])
     processed_data = processed_data.dropna(subset=["Synopsis"])
+    processed_data["intersection"] = processed_data["intersection"].str.replace("19: Maternal, Child", "19: Maternal Child")
+    processed_data["intersection"] = processed_data["intersection"].str.replace("24: Data Science, Computational", "24: Data Science Computational")
+    processed_data["Tagger 1 only"] = processed_data["Tagger 1 only"].str.replace("19: Maternal, Child", "19: Maternal Child")
+    processed_data["Tagger 1 only"] = processed_data["Tagger 1 only"].str.replace("24: Data Science, Computational", "24: Data Science Computational")
+    processed_data["Tagger 1 only"] = processed_data["Tagger 1 only"].str.replace("12: Brain Cells, Circuits", "12: Brain Cells Circuits")
+    processed_data["Tagger1_tags"] = processed_data.apply(
+        lambda x: x['intersection'].strip("[]").replace("'","").split(', ') + x['Tagger 1 only'].strip('[]').split(', '),
+        axis=1
+    )
     processed_data = processed_data.groupby('Grant_ID').agg({
         'Sciencetags': lambda x: ",".join(x),
         'Title': lambda x: x.iloc[0],
         'Synopsis': lambda x: x.iloc[0],
         'Lay Summary': lambda x: x.iloc[0],
         'Qu.': lambda x: x.iloc[0],
         'Scheme': lambda x: x.iloc[0],
-        'Team': lambda x: x.iloc[0]
+        'Team': lambda x: x.iloc[0],
+        'Tagger1_tags': lambda x: x.iloc[0]
```

Comment:
also i think you might want to add 'intersection' and 'tagger 1 only' to the output?

Reply:
what do you mean? i add them here, as there are multiple duplicate rows in the data. Kirstin broke the data to one row per tag for the tags with all other information duplicated. so all i am doing here is choosing one of the rows.

Reply:
ah yeh sorry I didnt notice that you use them to create "Tagger1_tags"